packages/eslint-plugin/docs/rules/prefer-regexp-exec.md

```diff
@@ -0,0 +1,38 @@
+# Enforce to use `RegExp#exec` over `String#match` (prefer-regexp-exec)
+
+`String#match` is outperformed by `RegExp#exec` and both work in the same way, except for /g flag.
```

Comment:
Would be useful to link to the source of that statement. (see https://github.com/typescript-eslint/typescript-eslint/issues/283)

Reply:
```suggestion
`RegExp#exec` is faster than `String#match` and both work the same when not using the `/g` flag.
```